modules/base/processors/imagesource.cpp

```diff
@@ -53,7 +53,8 @@ ImageSource::ImageSource(InviwoApplication* app, const std::string& file)
     , file_("imageFileName", "File name", file, "image")
     , imageDimension_("imageDimension_", "Dimension", ivec2(0), ivec2(0), ivec2(10000), ivec2(1),
                       InvalidationLevel::Valid, PropertySemantics("Text")) {
-
+    // Remove default data created by ImageOutport
+    outport_.setData(static_cast<Image*>(nullptr));
```

Comment:
should this be the default then?

Reply:
That would probably have quite an impact on existing code so it would need to be tested. Separate issue and PR IMHO